packages/www/components/WebhookDetails/index.tsx

```diff
@@ -26,11 +29,39 @@ import {
 import { useApi } from "hooks";
 import DeleteDialog from "../WebhookDialogs/DeleteDialog";
 import LogsContainer from "./LogsContainer";
+import { Webhook } from "@livepeer.studio/api";
 
-const StyledPencil = styled(Pencil1Icon, {
-  mr: "$1",
-  width: 12,
-  height: 12,
+const eventOptions: Webhook["events"] = [
```

Comment:
How to clear the filter "events". I don't see any option here.

<img width="239" alt="image" src="https://github.com/livepeer/studio/assets/2834997/5601ed00-eab9-4e7f-ac10-4c1e1fd519d7">